test/puppetlabs/puppetdb/http/aggregate_event_counts_test.clj

```diff
@@ -91,7 +91,48 @@
          ["=" "certname" "foo.local"]
          ["<=" "report_receive_time" current-time-str]
          ["<=" "run_start_time" current-time-str]
-         ["<=" "run_end_time" current-time-str])))
+         ["<=" "run_end_time" current-time-str]))
+
+  (store-example-report! (:basic2 reports) (coerce/to-string (now)))
+
+  (are [result query] (= result
+                         (query-result method endpoint query
+                                       {:summarize_by "resource"
+                                        :distinct_resources true
+                                        :distinct_start_time 0
+                                        :distinct_end_time (now)}))
```

Comment:
Hi @wkalt , if we are going to have to specify start/end times for latest_report? query, is it acceptable then just to use `0` and `now` as the time range as you have done here without fear of fiery explosion?